src/api/auth/Dockerfile

```diff
@@ -1,12 +1,50 @@
-FROM node:lts-alpine as base
-RUN apk add dumb-init
-WORKDIR /app
-COPY --chown=node:node ./package.json ./
+## Base ########################################################################
+# Use a larger node image to do the build for native deps (e.g., gcc, pytyhon)
+FROM node:lts as base
```

Comment:
Is this better than using alpine as base and installing the dependencies that we need? I'm asking because we're not only downloading 2 versions of node for this (not the best in terms of build time), but also the full version (node: lts) we might get a lot more than just the dependencies we need.

Reply:
It's pretty common to use this approach.  True, you have to download 2 base layers, but they will get cached and re-used over and over again.  Then you can have a full environment for creating the release, but run it on a much smaller layer.

Reply:
> but they will get cached and re-used over and over again

Not for staging and production, that's why I mentioned build time. We use fresh everything for every deployment, in case there are bug fixes that have been released upstream. I've seen the 1 layer + dependencies in production more often than the 2 layers approach, but our build time is not crazy long so maybe it's ok?

Reply:
We're moving toward building all the images and pushing to a registry (cc @TDDR), so this will become a non-issue, since the build won't happen on staging/production once it's done.  Until then, I don't think this is a huge problem.